build.bat

```diff
@@ -0,0 +1,16 @@
+rmdir /q /s import-mailbox-to-gmail
+rmdir /q /s import-mailbox-to-gmail-64
+rmdir /q /s build
+rmdir /q /s dist
+del /q /f import-mailbox-to-gmail-%1-windows.zip
+del /q /f import-mailbox-to-gmail-%1-windows-x64.zip
+
+c:\python27-32\scripts\pyinstaller -F --distpath=import-mailbox-to-gmail import-mailbox-to-gmail.spec
```

Comment:
Can you make the paths (e.g. `c:\python27-32` and 7zip's path) in central variables?